src/UnityExtension/Assets/Editor/GitHub.Unity/UI/SettingsView.cs

```diff
@@ -98,25 +99,32 @@ private void Setup()
                 return;
             }
 
+            UpdateRemote();
+        }
+
+        private void UpdateRemote()
+        {
             var currentRemote = Repository.CurrentRemote;
-            if (!currentRemote.HasValue && String.IsNullOrEmpty(repositoryRemoteName))
+            if (!currentRemote.HasValue)
             {
-                repositoryRemoteName = "origin";
-                repositoryRemoteUrl = "";
+                repositoryRemoteName = DefaultRepositoryRemoteName;
+                repositoryRemoteUrl = string.Empty;
+
                 Repository.OnActiveRemoteChanged += Repository_OnActiveRemoteChanged;
```

Comment:
Isn't this something that we always want to do regardless of whether there's a currently active remote or not?